packages/react-dom/src/test-utils/ReactTestUtils.js

```diff
@@ -456,7 +456,12 @@ function makeNativeSimulator(eventType) {
   };
 }
 
-Object.keys(topLevelTypes).forEach(function(eventType) {
+const eventKeys = [].concat(
+  Object.keys(topLevelTypes),
+  Object.keys(mediaEventTypes),
+);
```

Comment:
We _could_ use `EventPluginRegistry.eventNameDispatchConfigs` for SimulateNative, but it:

1. Does not include `invalid`, `mouseEnter`, `mouseLeave`, `reset`, `select`, or `submit`.
2. Adds `selectionChange`, `textInput`

Curiously, this is the case for `ReactTestUtils.Simulate`. Is this correct?

Reply:
Whatever’s currently there is “correct” in the sense that we shouldn’t change it now, and we should remove SimilateNative altogether in 17.